src/Entities/Player.cpp

```diff
@@ -65,6 +65,7 @@ cPlayer::cPlayer(cClientHandlePtr a_Client, const AString & a_PlayerName) :
 	m_GameMode(eGameMode_NotSet),
 	m_IP(""),
 	m_ClientHandle(a_Client),
+	m_IsFrozen(false),
```

Comment:
Hmm. Perhaps it would be wiser to have an enum. `m_FrozenState { None, Manual, Auto }`